cabinet/GoogleDrive.md

```
@@ -0,0 +1,280 @@
+  abs_path     TEXT NOT NULL,     -- absolute local path (Phase 1)
+  folder_name  TEXT NOT NULL,
+  enabled      INTEGER DEFAULT 1,
+  added_at     TEXT NOT NULL
+);
+```
+
+### Tree integration
+
+Mounts appear as a separate root section below local documents:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)